docs/mkdocs.yml

```diff
@@ -149,7 +149,20 @@ nav:
       - DJL Benchmark: 'docs/serving/benchmark/README.md'
   - DJL Serving:
       - Why DJL Serving?: 'docs/serving/README.md'
-      - Large Model Inference: 'docs/serving/serving/docs/large_model_inference.md'
+      - Large Model Inference:
+            - 'docs/serving/serving/docs/large_model_inference.md'
+            - 'docs/serving/serving/docs/lmi/configurations_large_model_inference_containers.md'
```

Comment:
We should remove duplicated link from configuration section

Reply:
Removed it